Makefile

```diff
@@ -46,15 +46,23 @@ git-diff :
 # op-ledger :;  forge script script/DeployArbitrumPayload.s.sol:ArbCaps --rpc-url arbitrum --broadcast --legacy --ledger --mnemonic-indexes ${MNEMONIC_INDEX} --sender ${LEDGER_SENDER} --verify -vvvv
 # op-pk :;  forge script script/DeployArbitrumPayload.s.sol:ArbCaps --rpc-url arbitrum --broadcast --legacy --private-key ${PRIVATE_KEY} --verify -vvvv
 
-usdt-strategy :;  forge script script/DeployUSDTV3Utils.s.sol:DeployUSDTStrategy --rpc-url mainnet --broadcast --legacy --ledger --mnemonic-indexes ${MNEMONIC_INDEX} --sender ${LEDGER_SENDER} --verify --etherscan-api-key ${ETHERSCAN_API_KEY_MAINNET} -vvvv
+# usdt-strategy :;  forge script script/DeployUSDTV3Utils.s.sol:DeployUSDTStrategy --rpc-url mainnet --broadcast --legacy --ledger --mnemonic-indexes ${MNEMONIC_INDEX} --sender ${LEDGER_SENDER} --verify --etherscan-api-key ${ETHERSCAN_API_KEY_MAINNET} -vvvv
```

Comment:
doesn't matter now, but plz don't touch stuff irrelevant to your proposal in upcoming prs.
this will just create unnecessary conflicts & make this pr harder to review.